pyconbalkan/speaker/migrations/0008_AddSpeakerExcerpt.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.0.5 on 2018-06-20 06:07
```

Comment:
Not a big deal but naming convention for file names is all-lowercase with underscores. The same applies to other PR with migration.

Reply:
No problem, I'll fix that in both PRs and push the change.